erpnext_shipping/erpnext_shipping/shipping.py

```diff
@@ -197,6 +362,37 @@ def print_shipping_label(shipment: str):
 			content = sendcloud.download_label(label_url)
 			file_url = save_label_as_attachment(shipment, content, i)
 			shipping_label.append(file_url)
+	elif service_provider == SHIPROCKET_PROVIDER:
+		shipping_label = []
+		shiprocket = ShiprocketUtils(company=pickup_company)
+		content = shiprocket.generate_label(shipment_id, shipment)
+		file_url = get_shipping_label(content)
```

Comment:
Missing import for this function `get_shipping_label`.

Appreciate your efforts in refactoring. Please test the refactored code thoroughly once.